streaming/stream_plan.cc

```diff
@@ -35,7 +35,7 @@ stream_plan& stream_plan::transfer_ranges(inet_address to, sstring keyspace, dht
 stream_plan& stream_plan::transfer_ranges(inet_address to, sstring keyspace, dht::token_range_vector ranges, std::vector<sstring> column_families) {
     _range_added = true;
     auto session = _coordinator->get_or_create_session(_mgr, to);
-    session->add_transfer_ranges(keyspace, std::move(ranges), std::move(column_families));
+    session->add_transfer_ranges(keyspace, std::move(ranges), column_families);
```

Comment:
I don't get this change. What was wrong with moving `column_families`? It's a local parameter in `transfer_ranges`, we don't use it later, so we can `move` it.